tests/testdata/mesh/quad_flower_to_edit_expected.2dm

```diff
@@ -3,8 +3,12 @@ ND 1 2000 2000 200
 ND 2 2500 2500 800
 ND 3 2000 3000 200
 ND 4 1000 3000 200
-ND 5 1500 1500 800
-ND 6 700 1750 0
+ND 5 1500 1500 -200
```

Comment:
doesn't test cases output pictures need to be updated too if you change this file?

Reply:
this file is not used for testing rendering